### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'items' from traitlets package

### ISSUE THREAD

**Issue by sir-dio** [jupyter/nbgrader#1244]
### Operating system
Ubuntu 18.04.3 LTS

### `nbgrader --version`
Python version 3.7.4 (default, Sep  2 2019, 20:47:34) 
[GCC 7.4.0]
nbgrader version 0.7.0.dev

### `jupyterhub --version`
1.0.0

### `jupyter notebook --version` 
6.0.1

I am setting up nbgrader on a new server, and there is a problem with the extensions. When I log in as an instructor and open the Courses tab in the notebook, the following error is displayed (full traceback from the console):

```
File "/srv/nbgrader/nbgrader/nbgrader/server_extensions/course_list/handlers.py", line 173, in get
        config = self.load_config()
      File "/srv/nbgrader/nbgrader/nbgrader/server_extensions/course_list/handlers.py", line 53, in load_config
        full_config.merge(config)
      File "/usr/local/share/dspython-venv/lib/python3.7/site-packages/traitlets/config/loader.py", line 185, in merge
        for k, v in other.items():
    AttributeError: 'tuple' object has no attribute 'items'
```

When I was testing everything couple of weeks ago on another machine, everything worked perfectly. Now, repeating the same steps, this happens.

**Comment by BertR** [comment]
HI @sir-dio , this is a known issue and there is a fix waiting for review here: https://github.com/jupyter/nbgrader/pull/1239

**Comment by jhamrick** [comment]
Closed by #1245
